tests/test_git.py

```diff
@@ -868,7 +868,7 @@ async def test_git_ssh(
     )
 
     async with sftp.open("test-repo.git/refs/heads/master") as fobj:
-        assert (await fobj.read()).strip() == rev
+        assert (await fobj.read(-1, 0)).strip() == rev
```

Comment:
[`SFTPFileProtocol.read`](https://github.com/ronf/asyncssh/blob/ce5b41abe30b399d86af1d04a18d092617cda23c/asyncssh/sftp.py#L217) requires `read` and `offset` to be passed explicitly. This is most likely a bug in typings in asyncssh, but this does not hurt us that much, so just fixing it here.